src/controllers/programs-controller.js

```diff
@@ -134,18 +186,21 @@ router.post("/", async (req, res) => {
         const tuitionFee = req.body.programTuition;
 
         if (!name || !programCode || !universityName || !prerequisite || !tuitionFee) {
-            return res.status(400).json({ error: "To create a new program, name, program code, university name, and prerequisites are required." });
+            // Translation: To create a new program the following information is required: name, program code, university name, prerequisites, and tuitionFee
+            return res.status(400).json({ error: "새로운 프로그램을 등록하기 위해서 다음 정보가 필요합니다: 이름, 학과코드, 자격요건, 학비" });
         }
 
         const invalidPrerequisites = prerequisite.filter(p => !["Math3B", "Math4", "Math5", "Physics1A", "Physics2", "Chemistry1", "Chemistry2", "Biology1", "Biology2", "Science2", "Civics1B", "History1B", "SpecialRequirement"].includes(p));
         if (invalidPrerequisites.length > 0) {
-            return res.status(400).json({ error: "Invalid prerequisites: " + invalidPrerequisites.join(", ") });
+            // Translation: Invalid prerquisite(s)
+            return res.status(400).json({ error: "존재하지 않는 자격요건: " + invalidPrerequisites.join(", ") });
         }
 
         const existingProgram = await Programs.findOne({ name: name });
 
         if (existingProgram) {
-            return res.status(409).json({ error: "The following program is already registered in the system." });
+            // Translation: The following program is already registered in the system
+            return res.status(409).json({ error: "다음 프로그램은 이미 시스템에 등록되어 있습니다." });
         }
```

Comment:
## Possible Problem
Though this would work if the administrator sends in a university in the exact same name, the same program can be expressed in several names (Software Engineering and Management, 소프트웨어과 etc.). This would be problematic in the future
## Possible Solution
Make a dictionary or something similar to avoid duplicate programs from being registered to our DB.

Reply:
This is a valid argument but as there are more things to work on for now, this will be addressed later on.